### PR TITLE
Feature/ID 04/implement search criteria

### PR DESCRIPTION
**Pull request by fabian-romero-jalafund**

### CODE REVIEW

File: SearchCriteria/Books/AuthorSearchCriteria.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿
```
Comment (gabriela-garcia-jalafund) [inline SearchCriteria/Books/AuthorSearchCriteria.cs]: Delete this space, just like you did in the other classes, good job.

**Review by gabriela-garcia-jalafund — APPROVED** [review]

**Review by gabriela-garcia-jalafund — APPROVED** [review]

**Comment by gabriela-garcia-jalafund** [comment]
image
you got the wrong ID, you are 05

File: SearchCriteria/Books/AuthorSearchCriteria.cs (hunk 1 above)
Comment (fabian-romero-jalafund) [reply SearchCriteria/Books/AuthorSearchCriteria.cs]: Already resolved, thanks

**Comment by fabian-romero-jalafund** [comment]
> image you got the wrong ID, you are 05

I will close this PR and open a new one with the correct ID, thanks

**Pull request closed without merging**